lib/fs.js

```diff
@@ -470,7 +480,7 @@ function readFileAfterRead(err, bytesRead) {
 
 function readFileAfterClose(err) {
   var context = this.context;
-  var buffer = null;
+  var buffer;
```

Comment:
@mscdex I'm guessing that it was put there as an optimization (so there is always a value here) - I don't see why it would make things faster to assign to null in this particular case though.